Formula/phplint.rb

```diff
@@ -21,6 +21,8 @@ class Phplint < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ecb11516875f096c647e254ef2451687ead874112397779abdb1afeafd8e0563"
   end
 
+  deprecate! date: "2020-11-26", because: "only supports deprecated PHP versions"
```

Comment:
Deprecate today? Better for scripting automated disables.

Reply:
Better not, since php@7.4 will be disabled at the end of this year.